src/imars3d/backend/workflow/schema.json

```diff
@@ -4,44 +4,57 @@
   "type": "object",
   "properties": {
     "facility": {
-      "type": "string"
+      "type": "string",
+      "enum": ["HFIR", "SNS"],
+      "description": "Facility for the measurment"
     },
     "instrument": {
-      "type": "string"
+      "type": "string",
+      "enum": ["CG1D", "SNAP"],
+      "description": "Instrument for the measurment"
     },
     "ipts": {
-      "type": "string"
+      "type": "string",
+      "description": "The full IPTS identifier for the measurement"
     },
     "name": {
-      "type": "string"
+      "type": "string",
+      "description": "Rememberable name for the measurement"
     },
     "workingdir": {
-      "type": "string"
+      "type": "string",
+      "description": "Directory to write intermediate results when requested"
     },
     "outputdir": {
-      "type": "string"
+      "type": "string",
+      "description": "Directory to write final results"
     },
     "tasks": {
       "type": "array",
       "minItems": 1,
+      "description": "Each task is a step in the tomographic reconstruction",
```

Comment:
can we change the `minItems` to 0 here?
to use the engine that based off this schema (the validation happens during init instead of running) in interactive reconstruction, we need to be able to create an instance of the engine that is instantiated with a almost blank config.